python/src/nanoarrow/schema.py

```diff
@@ -481,7 +499,7 @@ def null(nullable: bool = True) -> Schema:
 
     >>> import nanoarrow as na
     >>> na.null()
-    Schema(NULL)
+    <Schema> na
```

Comment:
This "na" is defined by the nanoarrow C sources? Because I think using "null" is 1) much clearer for the user, and 2) more in alignment with the actual spec